iamproperty/iamproperty.github.io
# Basic Card deck

## Summary of Changes:
1. Card deck component
2. Card component
3. New css
4. Unit tests/Visual tests

## Rationale
A card deck component can group together and highlight a series of links.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/card-deck

## Ticket(s)
DS-49

## Checklist

## Notes
This is just a basic card deck, further branches are going to be expanding upon them.
